src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+export const PeopleTable: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [loader, setLoader] = useState<boolean>(false);
+  const [error, setError] = useState<boolean>();
+  const { slug } = useParams();
+
+  const getPeopleFromServer = async () => {
+    setLoader(true);
+    const peopleFromServer = await getPeople();
+
+    setPeople(peopleFromServer);
+    setLoader(false);
+  };
+
+  useEffect(() => {
+    try {
+      getPeopleFromServer();
+    } catch {
+      setError(true);
+    }
+  }, []);
+
+  const findParentByName = (parentName: string | null) => {
+    return people?.find(
+      person => person.name === parentName || person.name === parentName,
+    );
+  };
+
+  return (
+    <div className="container">
+      <h1 className="title">People Page</h1>
+
+      {error && (
+        <p data-cy="peopleLoadingError" className="has-text-danger">
+          Something went wrong
+        </p>
+      )}
+
+      <div className="block">
+        <div className="box table-container">
+          {loader ? (
+            <Loader />
+          ) : (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
```

Comment:
When you show your table, you should also check if it has any people in it. So better to separate loader and table with people
```suggestion
          {loader && (
            <Loader />
          )}
          
          {!people?.length && (
                  <p data-cy="noPeopleMessage">
                    There are no people on the server
                  </p>
                )}
          
          {!loader && people.length > 0 && (
            <table
              data-cy="peopleTable"
              className="table is-striped is-hoverable is-narrow is-fullwidth"
            >
```